pkg/exporter/ovs.go

```diff
@@ -152,9 +180,14 @@ func (m *OvsMonitor) handleOvsUpdates(updates ovsdb.TableUpdates) {
 					m.collectorCache.AddIface(generateInterface(row.New))
 				}
 				if table == "Bridge" {
-					if FilterBridge(row.New.Fields["name"]) {
+					if FilterBridge(row.New.Fields["name"], datapath.POLICY_BRIDGE_KEYWORD) ||
+						FilterBridge(row.New.Fields["name"], datapath.CLS_BRIDGE_KEYWORD) {
```

Comment:
Does the uplinkBridge need to be filtered ?

Reply:
uplinkBridge is filtered in following function FilterSingleBridge()